### PR TITLE
Change UI messages to refer to HiredFiredPro

### LINKED ISSUE #154: UI messages still refer to addressbook
change all mentions of addressbook to hiredfiredpro in the UI messages

### PR DESCRIPTION
**Pull request by SQ77**
closes #154

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/ClearCommand.java [hunk 1]
```diff
@@ -6,12 +6,12 @@
 import seedu.address.model.Model;
 
 /**
- * Clears the address book.
+ * Clears HiredFiredPro.
```
Comment (bigismols) [inline src/main/java/seedu/address/logic/commands/ClearCommand.java:9]: Should this be something like "Cleared list of candidates"? Or maybe candidate list?

File: src/main/java/seedu/address/logic/commands/DeleteCommand.java [hunk 2]
```diff
@@ -12,18 +12,18 @@
 import seedu.address.model.person.Person;
 
 /**
- * Deletes a person identified using it's displayed index from the address book.
+ * Deletes a person identified using it's displayed index from HiredFiredPro.
```
Comment (bigismols) [inline src/main/java/seedu/address/logic/commands/DeleteCommand.java:15]: Same for this one and other changes too

File: src/main/java/seedu/address/logic/commands/SortCommand.java [hunk 3]
```diff
@@ -9,22 +9,22 @@
 import seedu.address.model.person.Person;
 
 /**
- * Represents a command to sort the person list by interview score.
+ * Represents a command to sort the candidate list by interview score.
```
Comment (bigismols) [inline src/main/java/seedu/address/logic/commands/SortCommand.java:12]: Might be better to leave this as person because it is a list of Person objects

**Review by bigismols — COMMENTED** [review]
Requested some suggestions.

File: src/main/java/seedu/address/logic/commands/SortCommand.java (hunk 3 above)
Comment (SQ77) [reply src/main/java/seedu/address/logic/commands/SortCommand.java:12]: I changed it to candidate to be consistent with the displayed UI messages

**Pull request merged**